spec/rspec_puppet_facts_spec.rb

```diff
@@ -319,7 +317,7 @@
       end
 
       it 'returns a fact set for the specified release' do
-        expect(factsets).to match('redhat-9-x86_64' => include(operatingsystemmajrelease: '9'))
+        expect(factsets['redhat-9-x86_64'][:os]['release']['major']).to eql('9')
```

Comment:
this is very ugly. It works, but there's probably a better way to write this?

Reply:
updated it to use proper rspec stuff. thanks to
* https://rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers:a_hash_including and 
* https://www.rubydoc.info/gems/rspec-expectations/RSpec/Matchers#include-instance_method
* https://stackoverflow.com/questions/40325820/rspec-partial-match-against-a-nested-hash